src/WebServer.Logging/ILogger.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Restup.WebServer.Logging
+{
+    public interface ILogger
```

Comment:
I personally like an overload with a Func<string> parameter. This will allow us generate trace outputs which otherwise would inflict performance penalties.